andela/bp-esa-frontend
# Ft docker file for deployment

#### What does this PR do?
Add a DockerFile to the repository for deployment

#### Description of Task to be completed?
Creation of app image for deployment to GCP

#### How should this be manually tested?
To test ensure you have docker installed
1. Clone the repository and switch to this branch (git checkout ft-docker-file-for-deployment)
2. Run in your shell docker build -t esa/frontend:latest .
3. Run docker run --rm  -it  -p 8088:80 esa/frontend:latest
4. On browser, goto url  http://localhost:8088 to view the served file

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Postman Documentation

N/A
#### Screenshots (If applicable)
N/A
